Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -53,22 +42,33 @@ spec:
 
     resources:
       limits:
-        cpu: "500m"
-        memory: "1024Mi"
+        cpu: "3"
+        memory: "8Gi"
       requests:
-        cpu: "500m"
-        memory: "1024Mi"
+        cpu: "3"
+        memory: "8Gi"
 
-    priorityClassName:
+    priorityClassName: ""
 
     logLevel: NIV_INFO
+
+  security:
```

Comment:
nit: How about moving the `security` section to the top of the spec? It seems more important than the placement, resources, and other settings.

Reply:
Is it more important than the spec for the number of servers though? I still think this might be the more logical place given that the security is optional.

Reply:
I see now, everything else is under the `server` section, so makes sense to leave it here.